modules/client/src/connext.ts

```diff
@@ -900,10 +900,11 @@ export class ConnextClient implements IConnextClient {
   };
 
   public matchTx = (
-    givenTransaction: Transaction,
+    givenTransaction: Transaction | undefined,
     expected: CFCoreTypes.MinimalTransaction,
   ): boolean => {
     return (
+      givenTransaction &&
       givenTransaction.to === expected.to &&
```

Comment:
Will this work if `givenTransaction` is `undefined`?

Reply:
yessir